x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/TransactionFlyout/index.tsx

```diff
@@ -28,7 +28,7 @@ interface Props {
   onClose: () => void;
   transaction?: Transaction;
   errorCount: number;
-  traceRootDuration?: number;
+  traceRootTransactionDuration?: number;
```

Comment:
Perhaps drop `trace` prefix?
```suggestion
  rootTransactionDuration?: number;
```